exareme2/algorithms/flower/flower_data_processing.py

```diff
@@ -0,0 +1,94 @@
+import json
+import os
+from pathlib import Path
+from typing import List
+from typing import Optional
+
+import pandas as pd
+import pymonetdb
+import requests
+from pydantic import BaseModel
+from sklearn import preprocessing
+from sklearn.impute import SimpleImputer
+
+# Constants for project directories and environment configurations
+PROJECT_ROOT = Path(__file__).resolve().parents[3]
+
+
+class Inputdata(BaseModel):
+    data_model: str
+    datasets: List[str]
+    filters: Optional[dict]
+    y: Optional[List[str]]
+    x: Optional[List[str]]
+
+    def fetch_data(self, from_db=False) -> pd.DataFrame:
+        return self._fetch_from_db() if from_db else self._load_from_csv()
+
+    def _fetch_from_db(self) -> pd.DataFrame:
```

Comment:
Rename to `InputDataDto` and remove the methods.

It will be more clear without any logic inside.